mbed_tools_ci/utils/git_helpers.py

```diff
@@ -462,3 +554,76 @@ def is_current_branch_feature(self) -> bool:
         is_beta = current_branch == self.get_beta_branch()
         is_release = self.is_release_branch(current_branch)
         return not (is_master or is_beta or is_release)
+
+    @property
+    def uncommitted_changes(self) -> List[Path]:
+        """Gets list of uncommitted files.
+
+        Returns:
+             list of uncommitted
+        """
+        status = self.repo.git.status(
+            porcelain=True,
+            untracked_files=True
+
+        )
+        if not status:
+            return []
+
+        return [
+            Path(self.root).joinpath(
+                line.strip().split(' ')[-1]) for line in status.splitlines()
+        ]
+
+    def apply_uncommitted_changes(self, other_repo: Union[Path, 'GitWrapper']) -> None:
```

Comment:
From what I've seen this is only ever called with `GitWrapper`, can we get rid of `Path` type?

Reply:
done